cardano-launcher/configuration/launcher/launcher-config.demo.yaml

```diff
@@ -28,18 +28,18 @@ nodePath: cardano-node
 launcherLogsPrefix: ${XDG_DATA_HOME}/Daedalus/mainnet/Logs/
 walletArgs: []
 nodeDbPath: ${XDG_DATA_HOME}/Daedalus/mainnet/DB/
-updaterPath: ./cardano-launcher/test/testUpdater.sh
+updaterPath: bash
 workingDir: ./
 frontendOnlyMode: true
 nodeLogPath: null
-updaterArgs: []
+updaterArgs: ["main"]
```

Comment:
What is `main` here?

Reply:
That's just an argument used for manually testing update system using mocks. If you pass "main" as an argument, it'll echo some messages to the console.
https://github.com/input-output-hk/cardano-shell/blob/master/cardano-launcher/test/testUpdater.sh#L2